authors/apps/articles/models.py

```diff
@@ -2,7 +2,7 @@
 from authors.apps.authentication.models import User
 
 class Article(models.Model):
-    slug = models.CharField(max_length=120, unique=True)
+    slug = models.CharField(max_length=100, unique=True)
     title = models.CharField(max_length=100)
```

Comment:
Slug and article title cannot have the same length. A slug should be longer than the article title.

Reply:
Thank you, on it.

Reply:
Well done @AtamaZack